vllm/model_executor/layers/utils.py

```diff
@@ -110,29 +113,27 @@ def rocm_unquantized_gemm_impl(
     from vllm.platforms.rocm import on_gfx9
     k = weight.shape[1]
     m = weight.shape[0]
-    if x.is_contiguous():
```

Comment:
@cagrikymk  leave this contiguous check.  it will catch mllama.py issues with the non-contiguous tensor shape